### ISSUE TITLE
[Function] Assign to each concept the context of usage

### ISSUE THREAD

**Issue by pasqLisena** [DOREMUS-ANR/knowledge-base#43]
[Vocab on repo](https://github.com/DOREMUS-ANR/knowledge-base/blob/master/vocabularies/function.ttl) - [Table](https://docs.google.com/spreadsheets/d/1ina81P-i_5Y8vyv4mEaNoz1zTcIVQhF5vyhedgkGNFI/edit?usp=drive_web)

We need an easy way to understand, checking on the function (i.e. composer, conductor, ...), to which part of the music production chain (i.e. expr. creation, performance,...) attach the activity. It is required for the converters.

The field `Contexte d'utilisation (à mettre en scopenote)` is not usable because:
1. it is in French
2. too long and complex

I am more thinking about a field that contains a single flat value.
Not-exhaustive list:
- `composition`
- `performance`
- `performance plan`
- `editing`
- `recording`
- `broadcasting`
- `production`

We need to also understand in which property add it. I give some inputs, but honestly i am not happy with the following proposals:
- `skos:related`
- `mus:U52_has_context`
------
Tasks:
- [ ] create the list of contexts
- [ ] choose the right property
- [ ] implement

**Comment by pasqLisena** [comment]
Alternative: make an external table with this relationships

**Comment by rtroncy** [comment]
I think it would be better to not use an external table but to encode this information in the SKOS file describing the concept scheme, preferably using skos-only properties such as ```skos:scopeNote``` or simply ```skos:note```

**Comment by pasqLisena** [comment]
+1 for `skos:scopeNote`

**Comment by pasqLisena** [comment]
About the task 1, here an extract of the table [Suivi des Referentiels](https://docs.google.com/spreadsheets/d/1eRnULs1Ke9zgsYtGLC4qZ5LITGYEIHUvKkrntaVPeLY/edit#gid=0):

> Classe : M31 Actor's Function
> 4 référentiels : 
> - fonctions liées à un évènement de création de l'oeuvre F27 ou de création de l'expression F28 si l'expression est représentative 
> - fonctions liées à un évènement de création de l'expression F28 si l'expression n'est pas représentative 
> - fonctions liées à un Recording Event F29
> - fonctions liées à un Editing Event M29
> - fonctions liées à une Performance Individuelle M28
> - fonctions liées à un Publication Event F30

**Comment by rtroncy** [comment]
@ccecconi Why is it written "4 référentiels" if there are 6 bulleted points?

**Comment by ccecconi** [comment]
Good question.... I can't remember why we wrote "4 référentiels"... I think the information in this file is a little old. For exemple, I don't think the difference between the representative and the none-representative expression is still accurate. It may be better to distinguish musical expressions and none-musical expressions. Moreover, we need functions linked to F25 Performance plan. 

Something like : 
- F28 Expression creation (connected to F22 Self-contained Expression + type (Musical Work, Choregraphic Work, etc.))
- F28 Expression creation (connected to F25 Performance Plan)
- F29 Recording Event
- M29 Editing Event
- M28 Performance individuelle / M27 Foreseen Individual Performance
- F30 Publication Event

**Comment by pasqLisena** [comment]
Solution from @delahousse: use the `skos:Collection` for grouping those Concepts.

**Comment by pasqLisena** [comment]
Wanted collections: 
- Musical work
- Generic creative work
- F25 Performance Plan
- F29 Recording Event
- M29 Editing Event
- M28 Performance individuelle / M27 Foreseen Individual Performance
- F30 Publication Event

**Comment by rtroncy** [comment]
During the general assembly, we decided to rely on [SKOS collections](https://www.w3.org/TR/skos-primer/#seccollections) to model contexts. @pasqLisena will provide an initial set of contexts to be validated

**Comment by rtroncy** [comment]
@pasqLisena Can you provide short name for each collections?

**Comment by pasqLisena** [comment]
The collections can have those names:
- `musical_work`
- `creative_work`
- `performance_plan`
- `recording`
- `editing`
- `performance`
- `publication`

I am imagining something like:
```turtle
<http://data.doremus.org/vocabulary/function/collection/musical_work> a skos:Collection ;
    rdfs:label "musical_work";
    skos:member ....
```

**Comment by ccecconi** [comment]
I've some trouble to understand the expression "creative work". Do you mean every work which is not a musical one ? If so, is it ok for you if there are different types of "creative work" (choregraphic, lighting, etc.) or do you prefer to have a specific term for each type of work ?

**Comment by pasqLisena** [comment]
I prefer just one term. For me, the meaning is: create a new work, connect it to the musical one

**Comment by pasqLisena** [comment]
fixed in 60ae94a0970d774570827ffe0d04ca381c5c9d6f
